jeffreywallphd/OpenFinAL
# Fall2024 ba5200 team2

Feature: Peer Comparison (Stock Comparison)

Description
This feature allows users to:
Input stock symbols for comparison.
View metrics comparison for the entered stock symbols.
Export the comparison as a CSV or PDF file.


Changes Made
Implemented stock symbol search functionality.
Added comparison table for metrics like price, P/E ratio, and market cap.
Integrated export options for CSV and PDF.

How to Test
Run the application locally.
Enter stock symbols (e.g., AAPL, TSLA, AMZN).
Verify the comparison table displays metrics for the stocks.
Test exporting:
Click "Export to CSV" to download the comparison as a CSV file.
Click "Export to PDF" to download the comparison as a PDF file.
